draft-todo-yourname-protocol.md

```diff
@@ -24,6 +24,13 @@ abbrev: "TODO - Abbreviation"
 category: info
 
 docname: draft-todo-yourname-protocol-latest
+submissiontype: IETF
+stand_alone: yes
+ipr: trust200902
+pi: [toc, sortrefs, symrefs]
+number:
+date:
+consensus:
```

Comment:
I don't want to include too much noise here.  Most of these can be omitted safely.

If the goal is to have these available so that you don't have to go searching for the right key name, then I can see a use for these three (consensus should be "yes"), not the others.

Reply:
The goal here was to make it easier to fill those in during AUTH48 by not having to search for the key name. Does this look better?